cloud-tests-infrastructure/storage.tf

```diff
@@ -0,0 +1,27 @@
+module "storage" {
+  source = "git@github.com:leancodepl/terraform-common-modules.git//azure_blob_storage?ref=v0.1.0"
```

Comment:
The `leancodepl/terraform-common-modules` repo is private. Can we not reference it here and do most of the things manually?